app/views/billing_log_events/_table.html.haml

```diff
@@ -18,7 +18,21 @@
       %tr
         %td= I18n.l(log_event.event_time, format: :usa)
         %td= text(log_event.locale_tag)
-        %td= object_tag(log_event.object, class: "billing-logs__object")
+        %td.billing-logs__object
+          = object_tag(log_event.object, class: "billing-logs__object")
+          - if log_event.event_type == "review_orders_email" && log_event.has_orders?
+            %br
+            = render "billing_log_events/collapsible_list",
+              items: log_event.order_ids,
+              visible_count: 5,
+              id_prefix: "log-event-orders",
+              unique_id: log_event.id
+          - if log_event.has_reconciliation_data?
+            %br
+            = render "billing_log_events/collapsible_notes",
+              notes: log_event.all_reconciliation_notes,
+              id_prefix: "log-event-notes",
+              unique_id: log_event.id
```

Comment:
I think we can refactor this into something like

```
render_if_exist "billing_log_events/#{loggable_type.underscore}_#{event_type}"
```

and move off the logic to the partials